llvm_passes/CustomTensorOperatorInstSelector.cpp

```diff
@@ -0,0 +1,277 @@
+#include "llvm/IR/Instructions.h"
+#include "llvm/Support/CommandLine.h"
+
+#include "FIInstSelector.h"
+#include "FICustomSelectorManager.h"
+#include "Utils.h"
+
+#include "FICustomSelectorManager.h"
+
+#include <vector>
+#include <string>
+#include <algorithm>
+#include <cctype>
+#include <unordered_map>
+#include <cassert>
+
+using namespace llvm;
+
+namespace llfi {
+
+
+static cl::list< std::string > layerNo("layerNo",
+         cl::desc("Layer Number in which you want to inject bitflip faults. Pass 0 for injecting faults in all the layers.\n Semi-colon seperated values. Example: 1;0;2"),
```

Comment:
For LLVM source code, it would be preferable to keep all the lines within 80 columns.